libselinux/patches/selabel-digest-uninit.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/utils/selabel_digest.c b/utils/selabel_digest.c
+index 47aad21f..c574d3fd 100644
+--- a/utils/selabel_digest.c
++++ b/utils/selabel_digest.c
+@@ -65,7 +65,7 @@ int main(int argc, char **argv)
+ 	size_t digest_len, i, num_specfiles;
+ 
+ 	char cmd_buf[4096];
+-	char *cmd_ptr;
++	char *cmd_ptr = NULL;
```

Comment:
To be upstreamed

Reply:
let's also add a link to the patch at the top of the file if it was from alpine or some mailing list

Reply:
Ah, not sure. Came up with those myself based on compiler errors :muscle: :brain: :)

If any of those has been posted already then will tag for sure

Reply:
Could not google this up. Marked one found in Gentoo as well. It's not exactly the same since I made my myself from the current tree, but the idea is the same.